app/components/role_type_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="mt-6 border-t border-gray-200 pt-6 space-y-8">
+  <div>
+    <h2 class="text-sm font-medium text-gray-500">Interested in roles</h2>
+    <ul role="list" class="mt-3 space-y-3">
+      <% role_types.each do |role| %>
```

Comment:
Following on from 3484888, this can now be updated to:

```suggestion
      <% developer_role_types.each do |role| %>
```

and the nested condition can be removed.

Reply:
Moving `RoleType` to a db-backed model cleaned this up a bit!

Reply:
I _think_ it should still be possible to remove the nested condition by using the method you've defined on the `RoleTypeComponent`: 

```suggestion
      <% selected_role_types.each do |role| %>
```